simple-wp-helpdesk/src/Rest/TicketController.php

```
@@ -227,9 +227,14 @@ public function update_item( $request ): WP_REST_Response {
 			swh_set_ticket_status( $id, $patch['status'] );  // bumps via B2.
 		}
 		if ( array_key_exists( 'assignee', $patch ) ) {
-			$assignee = (int) $patch['assignee'];
-			update_post_meta( $id, '_ticket_assigned', $assignee > 0 ? $assignee : '' );
+			$assignee     = (int) $patch['assignee'];
+			$old_assigned = swh_get_int_meta( $id, '_ticket_assigned_to' );
+			update_post_meta( $id, '_ticket_assigned_to', $assignee > 0 ? $assignee : '' );
 			TicketVersion::bump( $id );
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\SWH\Concurrency\TicketVersion' in method 'update_item'. (undefined)

(StaticAccess)